src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() {
```

Comment:
Header comment is necessary for a method to be overriden. It helps to inform others on what the desired behaviour is. Meanwhile, as mentioned during the tutorial, it is better to enforce the overriding in all subclasses for clarity. Think about how to do this elegantly. Hint: just one keyword added to the method signature will do.